_includes/manuals/2.4/3.6_upgrade.md

```diff
@@ -31,113 +52,136 @@ For more information about how to backup your instance head over to
 
 #### Step 2 - Perform the upgrade
 
-Before proceeding, it is necessary to shutdown the Foreman instance (e.g.
-`systemctl stop httpd foreman.service foreman.socket` or
-`systemctl stop apache foreman.service foreman.socket` usually).
-
-Now it's time to perform the actual upgrade.  This process if different
-depending on how you downloaded Foreman.  You only need to perform *one* of
-the following options.
-
-##### Step 2 (A) - RHEL package (RPM) and installer setups
-
+Before proceeding, it is necessary to shutdown the Foreman instance.
+
+<div class="upgrade_os upgrade_os_none">
+  <i>No operating system selected.</i>
+</div>
+<div class="upgrade_os upgrade_os_el7 upgrade_os_el8">
+{% highlight bash %}
+systemctl stop httpd foreman.service foreman.socket
```

Comment:
What is your thinking behind also stopping the socket?

Reply:
Dynflow should be stopped as well.

Reply:
> What is your thinking behind also stopping the socket?

That it was there before. Also, the socket can start the service so that keeps it stopped.

> Dynflow should be stopped as well.

Yes and I considered that but that wasn't there before either. I'll take a look at this.